src/test/index.ts

```diff
@@ -1,15 +1,18 @@
-import axios from 'axios';
-import {} from 'mocha';
-import { expect } from 'chai';
+import * as dotenv from 'dotenv';
+import { startApp } from '../app-setup';
 
-describe('GraphQL Hello Query', () => {
-  it('returns "Hello World"', async () => {
-    const query = `query { hello } `;
+import {} from 'mocha';
+import { dropDB } from '../data-source';
 
-    const request = await axios.post('http://localhost:3001/', { query });
+dotenv.config({ path: __dirname + '/../../test.env' });
```

Comment:
```suggestion
dotenv.config({ path: process.cwd() + '/test.env' });
```

Essa `process.cwd()` te retorna a raiz do projeto.